Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -311,46 +312,11 @@
         "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas",
         "XamlCodeFile": "InfiniteCanvas.bind",
         "Icon": "/SamplePages/InfiniteCanvas/InfiniteCanvas.png",
+        "ApiCheck": "Windows.UI.Xaml.Controls.ColorPicker",
```

Comment:
Is the control's min version RS3 then?  As the doc says RS1...  Seems like the doc may need to be updated.  Though if the color picker is the only reason, then maybe the control should do an API check and just not have the color picker on down-level platforms?  Thoughts?

Reply:
I created an issue for this #2162 . For now, I set the min to 16299

Reply:
Cool, we should have the `BadgeUpdateVersionRequired` on it for now and update the doc with the minimum until 2162 is done.

Reply:
Done